### DISCUSSION TITLE
Attach scanner to a custom HTML-element

### DISCUSSION THREAD

**Discussion by Uzenkov** [capawesome-team/capacitor-mlkit#81] [Q&A] [answered]
Good day guys. Is it possible to attach the "barcode-scanner-active" class to a custom HTML-element? I want to show the scanner in a 300x300 pixels div-block. I've tried the following: document.querySelector("#my-id")?.classList.add("barcode-scanner-active"), but no luck.

**Comment by robingenz** [comment, marked as answer]
The `barcode-scanner-active` class just hides all webview elements. However, this class is only an example from the documentation (see [Usage](https://capawesome.io/plugins/mlkit/barcode-scanning/#usage)) and is not automatically provided by the plugin.

You might be looking for #79.

**Reply by Uzenkov** [reply]
I got it, thank you for rapid answer!
